pkgs/top-level/python-packages.nix

```
@@ -6452,6 +6452,8 @@ in {
 
   ovh = callPackage ../development/python-modules/ovh { };
 
+  ovmfvartool = callPackage ../applications/virtualization/ovmfvartool { };
```

if it's going to be listed in python-packages.nix, then we should probably also move the file to `python-modules`
```suggestion
  ovmfvartool = callPackage ../development/python-modules/ovmfvartool { };
```